snippets/currentDate.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Ask your name and tells you the current date
+tags: browser, begginer
+---
+
+Ask a user to enter the name and returns current date
```

Comment:
- Declare n1 variable to save the user's name	 
  
 - Declare dayToday to save the current date	 
  
 - Use new Date() to get the current date and hour	 
  
- Declare a function to return the user's name and current date